### PR TITLE
Update schematic.md

### PR DESCRIPTION
**Pull request by microbit-pauline**
Link to V2 BOM & schematic and change of language

### CODE REVIEW

File: hardware/schematic.md [hunk 1]
```diff
@@ -13,21 +13,23 @@ review-with: jonnya
 
 ## Overview
 
-This page discusses the micro:bit schematic and Bill of Materials (BOM), which shows the electrical connections of the micro:bit and the components used in it.
+This page discusses the micro:bit schematic and Bill of Materials [BOM](https://github.com/microbit-foundation/microbit-v2-hardware/blob/main/V2/Bill.of.Materials-BBC-microbit_V2.0.0.csv), which shows the electrical connections of the micro:bit and the components used in it.
 
-The schematic is available from the BBC's [micro:bit hardware repository](https://github.com/bbcmicrobit/hardware)
+The micro:bit V1.3 and V1.5 schematic is available from the BBC's [micro:bit hardware repository](https://github.com/bbcmicrobit/hardware).
```
Comment (microbit-mark) [inline hardware/schematic.md:18]: We should probably add the CSS classes that distinguish the version numbers

```suggestion
The micro:bit <span class="v1">V1.3</span> and  <span class="v1">V1.5</span> schematic is available from the BBC's [micro:bit hardware repository](https://github.com/bbcmicrobit/hardware).
```

File: hardware/schematic.md [hunk 2]
```diff
@@ -13,21 +13,23 @@ review-with: jonnya
...
+
+The micro:bit V2 schematic is available from the Micro:bit Educational Foundation [microbit-v2-hardware repository](https://github.com/microbit-foundation/microbit-v2-hardware).
```
Comment (microbit-mark) [inline hardware/schematic.md:20]:
```suggestion
The micro:bit V2 schematic is available from the Micro:bit Educational Foundation [microbit-v2-hardware repository](https://github.com/microbit-foundation/microbit-v2-hardware).
```
```suggestion
The micro:bit  <span class="v2">V2</span> schematic is available from the Micro:bit Educational Foundation [microbit-v2-hardware repository](https://github.com/microbit-foundation/microbit-v2-hardware).
```

File: hardware/schematic.md [hunk 3]
```diff
@@ -13,21 +13,23 @@ review-with: jonnya
...
 
 - [V1.3](https://github.com/bbcmicrobit/hardware/blob/master/V1.3B/SCH_BBC-Microbit_V1.3B.pdf)
 - [V1.5](https://github.com/bbcmicrobit/hardware/blob/master/V1.5/SCH_BBC-Microbit_V1.5.PDF)
-- [V2 TBC](#)
+- [V2](https://github.com/microbit-foundation/microbit-v2-hardware/blob/main/V2/MicroBit_V2.0.0_S_schematic.PDF)
 
 ## V2 pinmap
 
-Whilst we work on the publication of the schematic for the latest revision, here is the pinmap and allocation of the nRF52833
+Below is the pinmap and allocation of the nRF52833, more information is available on the micro:bit V2 [schematic](https://github.com/microbit-foundation/microbit-v2-hardware/blob/main/V2/MicroBit_V2.0.0_S_schematic.PDF)
```
Comment (microbit-mark) [inline hardware/schematic.md:32]:
```suggestion
Below is the pinmap and allocation of the nRF52833, more information is available on the micro:bit  <span class="v2">V2</span> [schematic](https://github.com/microbit-foundation/microbit-v2-hardware/blob/main/V2/MicroBit_V2.0.0_S_schematic.PDF)
```

File: hardware/schematic.md (hunk 1 above)
Comment (microbit-pauline) [reply hardware/schematic.md:18]: I don't know what CSS classes are, so agree with your judgement

**Comment by microbit-mark** [comment]
Thanks, Pauline. This is great. I've added some suggestions just to add the CSS that we use to distinguish button numbers, so they render eg:

image

File: hardware/schematic.md (hunk 2 above)
Comment (microbit-pauline) [reply hardware/schematic.md:20]: Agree

File: hardware/schematic.md (hunk 3 above)
Comment (microbit-pauline) [reply hardware/schematic.md:32]: Agree

**Comment by microbit-mark** [comment]
Added CSS suggestions, so merging. Thanks @microbit-pauline 👍🏻

**Pull request merged**